setup.cfg

```diff
@@ -81,6 +81,51 @@ setup_requires =
     setuptools >= 38.4
     setuptools_scm
 
+[options.extras_require]
+io =
+    netCDF4
+    h5netcdf
+    scipy
+    pydap
+    zarr
+    fsspec
+    cftime
+    rasterio
+    cfgrib
+    ## Scitools packages & dependencies (for example: cartopy, cf-units) can be hard to install
+    # scitools-iris
+    ## Required by scitools-iris
+    # scitools-pyke
+
+accel =
+    scipy
+    bottleneck
+    numbagg
+
+parallel =
+    dask[array,dataframe]
```

Comment:
are `array` and `dataframe` the only packages which can be used together with `xarray`? If I'm not missing anything, we could put the entire package (`dask[complete]`) and `distributed` into `parallel`.